app/models/user.ts

```diff
@@ -71,7 +71,7 @@ export default class UserModel extends Model {
 
   // Our discounts currently only apply to yearly plans. Change this if we need to support other pricing frequencies.
   get activeDiscountForYearlyPlan(): PromotionalDiscountModel | null {
-    return this.activeDiscountFromAffiliateReferral || this.activeDiscountFromSignup;
+    return this.activeDiscountFromAffiliateReferral || this.activeDiscountFromSignup || this.activeDiscountFromStageTwoCompletion;
```

Comment:
Let's prioritize the stage 2 completion one over the signup one - it feels more personalized. 

- Affiliate referral is prioritized first since a user explicitly accepted an offer
- Stage 2 completion can be next since it's linked to an action the user took
- Signup can be last, it's less personalized than the two above

Reply:
@ryan-gang we still haven't changed the order here no?